### PR TITLE
Refresh the token periodically in case of a change

### PR DESCRIPTION
**Pull request by iNecas**

### CODE REVIEW

File: pkg/config/configobserver/configobserver.go [hunk 1]
```diff
@@ -50,6 +50,9 @@ func New(defaultConfig config.Controller, kubeClient kubernetes.Interface) *Cont
 
 func (c *Controller) Start(ctx context.Context) {
 	wait.Until(func() {
+		if err := c.retrieveToken(); err != nil {
+			klog.Warningf("Unable to retrieve initial token config: %v", err)
```
Comment (mfojtik) [inline pkg/config/configobserver/configobserver.go]: s/initial//

Comment (mfojtik) [reply pkg/config/configobserver/configobserver.go]: @smarterclayton should this propagate to operator status (unavailable/degraded=true ?)

Comment (iNecas) [reply pkg/config/configobserver/configobserver.go]: Done

Comment (iNecas) [reply pkg/config/configobserver/configobserver.go]: We'll get notified about that at the next attempt to actually use the pull secret.

**Comment by iNecas** [comment]
Bump @mfojtik?

**Pull request merged**